tests/phpunit/tests/post.php

```diff
@@ -1637,4 +1637,53 @@ function test_wp_resolve_post_date() {
 		$resolved_post_date = wp_resolve_post_date( $invalid_date, $invalid_date );
 		$this->assertFalse( $resolved_post_date );
 	}
+
+	/**
+	 * @ticket 52007
+	 */
+	function test_sticky_posts() {
+		$orig_sticky_posts_option = get_option( 'sticky_posts' );
+		delete_option( 'sticky_posts' );
```

Comment:
The test suite resets options between tests so you probably don't need these lines or the final update_option in each test.